school_library/setup/book.rb

```diff
@@ -0,0 +1,11 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] Please 🙏🏽 add a method for adding rentals. Link to the [example](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship). The book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.